### PR TITLE
[Tickets] Linux & SQL & Grep & JDBC & Jarvis Profile 

### PR DESCRIPTION
**Pull request by salaymeh**

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+# Introduction
```
Comment (emhuskic) [inline linux_sql/README.md:2]: Readme file should be descriptive in terms of what the application/script does, how to get it started and how to use it.

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+cmd=$1
+db_username=$2
+db_password=$3
+# we are checking if docker is running if its not start docker
+sudo systemctl status docker || sudo systemctl start docker
+#assing the status of the container to see if its running
+#container_status=$(docker container inspect -f '{{.State.Running}}' jrvs-psql)
+docker container inspect jrvs-psql
+container_status=$?
+#export PGPASSWORD=$db_password
```
Comment (emhuskic) [inline linux_sql/scripts/psql_docker.sh:13]: This should not be commented out.

File: linux_sql/scripts/psql_docker.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    fi
+    docker container "$cmd" jrvs-psql
+    exit 0
+    ;;
+  *)
+    echo "illegal command"
+    echo "Commands: start|stop|create"
+    exit 1
+    ;;
+  esac
```
Comment (emhuskic) [inline linux_sql/scripts/psql_docker.sh:58]: New line at the end of file is missing. It is a convention to have it.

File: linux_sql/sql/ddl.sql [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  "timestamp" TIMESTAMP NOT NULL,
+  host_id SERIAL NOT NULL,
+  memory_free INT,
+  cpu_idle INT,
+  cpu_kernel INT,
+  disk_io INT,
+  disk_available INT,
+  FOREIGN KEY(host_id) REFERENCES host_info(id)
+);
+INSERT INTO host_info (
```
Comment (emhuskic) [inline linux_sql/sql/ddl.sql:22]: Insert script should be separate from the database definition script (creating the tables etc).

**Review by emhuskic — CHANGES_REQUESTED** [review]

File: linux_sql/scripts/psql_docker.sh (hunk 2 above)
Comment (salaymeh) [reply linux_sql/scripts/psql_docker.sh:13]: Will fix this!!

File: linux_sql/sql/ddl.sql (hunk 4 above)
Comment (salaymeh) [reply linux_sql/sql/ddl.sql:22]: will fix this!!

**Comment by salaymeh** [comment]
all the changes for this pull request are in the [Ticktets] Monitoring Agent and SQL Queriesls

**Pull request closed without merging**